week2/homework/src/methods.js

```diff
@@ -0,0 +1,94 @@
+const fs = require('fs');
+const util = require('util');
+
+const readFile = util.promisify(fs.readFile);
+const writeFile = util.promisify(fs.writeFile);
```

Comment:
This variable is declared, but it is actually never used in this file. Tip: check the difference between how you use `readFile` and `writeFile`.

Reply:
done.